node/src/chain_spec.rs

```diff
@@ -193,5 +193,7 @@ fn testnet_genesis(
 			min_solar_trip_time_s: 1, // [s]
 			max_speed_mps: 1,         // [m/s] suggested would be 83m/s,
 		},
+
+		encointer_balances: EncointerBalancesConfig { fee_conversion_factor: 10_000 },
```

Comment:
I'd like a comment here to explain what this value means and how to calculate it.

Reply:
like:
		// for relative adjustment.
			// 100_000 translates 5uKSM to 0.01 CC if ceremony reward is 20 CC
			// lower values lead to lower fees in CC proportionally

Reply:
..and I therefore suggest to set the value to 100_000

Reply:
your unit tests in pallet repo use 100_000 that's why I changed it

Reply:
Sorry, my fingers were in the game there. Yes `100_000` should it be now.